lambda/main.go

```diff
@@ -127,17 +128,24 @@ func LambdaHandler(event NewRelease) error {
 	triggerWorkflowAllowList := strings.Split(os.Getenv("TRIGGER_WORKFLOW_ALLOW_LIST"), " ")
 	if shouldTriggerWorkflow(pulumiRepo, triggerWorkflowAllowList) {
 		log.Printf("Pulumi repo '%s' was found in the allow list. Triggering workflow.", pulumiRepo)
-
+		repoObj, resp, err := gitHubClient.Repositories.Get(context.Background(), "pulumi", pulumiRepo)
+		if resp.StatusCode != http.StatusOK {
+			log.Printf("Pulumi repo %s could not be found: %s", pulumiRepo, resp.Status)
```

Comment:
This should be `return fmt.Errorf()` - basically a panic.

Reply:
Nit: Also, I believe you may need to accommodate _any_ 2XX response from the API - docs would say for sure.

Reply:
I'm gonna go with the trust-and-yell-if-it-changes model on this one:
https://docs.github.com/en/rest/reference/repos#get-a-repository <-- returns a 200 OK